_observing-your-data/ad/security.md

```diff
@@ -25,6 +25,12 @@ The security plugin has two built-in roles that cover most anomaly detection use
 
 If these roles don't meet your needs, mix and match individual anomaly detection [permissions]({{site.url}}{{site.baseurl}}/security/access-control/permissions/) to suit your use case. Each action corresponds to an operation in the REST API. For example, the `cluster:admin/opensearch/ad/detector/delete` permission lets you delete detectors.
 
+### A note on alerts and fine-grained access control
+
+When a trigger generates an alert, the detector and monitor configurations, the alert itself, and any notification that is sent to a channel may include metadata describing the index being queried. By design, the plugin must extract data and store it as metadata outside of the index. OpenSearch Security’s document-level security (DLS) and field-level security (FLS) fine-grained access controls are designed to protect data in the index. But once data is stored outside the index as metadata, users with access to detector and monitor configurations, alerts, and their notifications will be able to view this metadata and possibly infer the contents and quality of data in the index, which would otherwise be concealed by DLS and FLS access control.
```

Comment:
```suggestion
When a trigger generates an alert, the detector and monitor configurations, the alert itself, and any notification that is sent to a channel, may include metadata describing the index being queried. By design, the plugin must extract the data and store it as metadata outside of the index. The OpenSearch document-level security (DLS) and field-level security (FLS) access controls are designed to protect the data in the index. But once the data is stored outside the index as metadata, users with access to the detector and monitor configurations, alerts, and their notifications, will be able to view this metadata and possibly infer the contents and quality of data in the index, which would otherwise be concealed by DLS and FLS access control.

To reduce the chances of unintended users viewing metadata that could describe an index, we recommend that administrators enable role-based access control and keep these kinds of design elements in mind when assigning permissions to the intended group of users. For more information, see [Limit access by backend role](#advanced-limit-access-by-backend-role).
```